app/models/setting/content.rb

```diff
@@ -127,8 +127,7 @@ def self.default_settings
                false, N_('Content View Dependency Solving Default')),
       self.set('dependency_solving_algorithm',
                N_("How the logic of solving dependencies in a Content View is managed. Conservative will only add " \
-               "packages to solve the dependencies if the package needed doesn't exist. Greedy will pull in the " \
-               "latest package to solve a dependency even if it already does exist in the repository."),
+               "packages to solve the dependencies if the package needed doesn't exist."),
                'conservative', N_('Content View Dependency Solving Algorithm'), nil,
                :collection => dependency_solving_options),
```

Comment:
I think we can just completely get rid of the setting altogether.  You might need a migration to delete the setting, need to test it without the setting defined